src/app/styles/skeletons.css

```diff
@@ -0,0 +1,49 @@
+@import "react-loading-skeleton/dist/skeleton.css";
+
+/*.react-loading-skeleton {*/
```

Comment:
I copied this from the dashbrd repo, commented out overrides as colors aren't defined yet.